zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/CollectionTest.java

```diff
@@ -1415,8 +1415,8 @@ public void populateReleaseQuietlyShouldAddLinksToReleasePageForCollectionConten
         collection.complete(publisher1Session, releaseJsonUri, recursive);
         collection.review(publisher2Session, releaseJsonUri, recursive);
 
-        ContentDetail cmdDetail = new ContentDetail("My CMD dataset", "/some/uri", PageType.API_DATASET_LANDING_PAGE);
-        FileUtils.write(collection.getReviewed().getPath().resolve("some/uri/data.json").toFile(),
+        ContentDetail cmdDetail = new ContentDetail("My CMD dataset", "some/uri/versions/test", PageType.API_DATASET_LANDING_PAGE);
+        FileUtils.write(collection.getReviewed().getPath().resolve("some/uri/versions/test/data.json").toFile(),
```

Comment:
I don't think we need to change the json path here, please revert

```suggestion
        FileUtils.write(collection.getReviewed().getPath().resolve("some/uri/data.json").toFile(),
```